packages/manager/apps/web/client/app/hosting/multisite/git-association/remove-confirmation-modal/remove-confirmation-modal.component.js

```diff
@@ -0,0 +1,12 @@
+import template from './remove-confirmation-modal.html';
+
+export default {
+  bindings: {
+    goToDeleteGitAssociation: '<',
+    alertId: '<',
+    goBack: '<',
+    serviceName: '<',
+    path: '<',
+  },
+  template,
```

Comment:
Didn't you forget to add the controller here ?
Or maybe it's the controller that is unecessary ?

Reply:
No controller necessary